packages/ui/src/components/va-alert/VaAlert.stories.ts

```diff
@@ -1,12 +1,217 @@
-import { defineComponent } from 'vue'
-import VaAlert from './VaAlert.demo.vue'
+import { VaAlert } from './'
+import { VaButton } from '../va-button'
+import { userEvent } from '../../../.storybook/interaction-utils/userEvent'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaAlert',
   component: VaAlert,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
+export const Default = {
+  args: {
+    description: 'Message',
+  },
+}
+
+export const Dense = {
+  args: {
+    dense: true,
+    description: 'Message',
+  },
+}
+
+export const Title = {
+  args: {
+    title: 'Title',
+    description: 'Message',
+  },
+}
+
+export const Center = {
+  args: {
+    center: true,
+    title: 'Title',
+    description: 'Message',
+  },
+}
+
+export const Icon = {
+  args: {
+    icon: 'Icon',
```

Comment:
This format is dangerous and I would avoid it for now. Somehow - what you pass to args - is not used as props, but as slots instead. You can compare to this story.

```
export const Icon = () => ({
  components: { VaAlert },
  template: `
    <va-alert icon="home" description="Message"/>
  `,
})
```

We probably have to declare something in meta (`export default`) for this format to work properly. Probably if component has both slot and prop - it prioritises slot. (wild guess)